common/djangoapps/student/models.py

```diff
@@ -1621,6 +1668,12 @@ def user_can_skip_entrance_exam(cls, user, course_key):
 class LanguageProficiency(models.Model):
     """
     Represents a user's language proficiency.
+
+    Note that we have not found a way to emit analytics change events by using signals directly on this
+    model or on UserProfile. Therefore if you are changing LanguageProficiency values, it is important
```

Comment:
Ah, I think I understand, the wording made me think that you weren't able to instrument UserProfile. Instead it's saying that you can't instrument LanguageProviciencies via UserProfile changes.